build/constants.yaml

```diff
@@ -456,6 +458,20 @@
   constants:
     dotnetcore-staging-runtime-versions:
       -
+    python-staging-runtime-versions:
+      -
+    node-staging-runtime-versions:
+      -
+    java-staging-runtime-versions:
+      -
+    php-staging-runtime-versions:
+      -
+    hugo-staging-runtime-versions:
+      -
+    ruby-staging-runtime-versions:
+      -
+    golang-staging-runtime-versions:
+      -    
   outputs:
```

Comment:
Could you please clarify why these constants do not have values assigned?

Reply:
I have the same question 🙂 My guess is to have them populated later? If so, we should be able to populate these when the value is available

Reply:
Can we assign a `string.Empty` to these constants instead of leaving them empty?